src/innhold/innhold-logikk-niva4.tsx

```diff
@@ -36,10 +37,11 @@ const InnholdLogikkNiva4 = ({
 
     const innsatsgruppeData: Data | null = React.useContext(InnsatsgruppeContext).data;
     const innsatsgruppe = innsatsgruppeData ? innsatsgruppeData.servicegruppe : null;
+    const isMicrofrontend = erMikrofrontend ? erMikrofrontend.toString() === 'true' : false;
 
     React.useEffect(() => {
         seVeientilarbeid(erSykmeldtMedArbeidsgiver, innsatsgruppe);
-        hotjarTrigger();
+        hotjarTrigger(isMicrofrontend);
```

Comment:
Jeg ga et lite intuitivt svar på Slack ang. dette på fredag. Sorry for det 😇

Det skal holde å gjøre funksjonskallet direkte som argument til funksjonen.

Altså `hotjarTrigger(erMikrofrontend());`

Reply:
ahhh, takk
Fikser, pusher og merger